lang/ui.nl.json

```diff
@@ -611,7 +611,7 @@
     "description": "Aria label for the redo button"
   },
   "reference-tab": {
-    "defaultMessage": "Referentie",
+    "defaultMessage": "Sommige van je wijzigingen zijn niet opgeslagen. Toch afsluiten?",
```

Comment:
We can't merge this change!

From

![image](https://user-images.githubusercontent.com/44397098/203091876-29f34f2a-5b0a-446a-9d87-d14f46405205.png)

To

![image](https://user-images.githubusercontent.com/44397098/203091985-06e9e23a-881e-4aab-9156-673d9f7887cf.png)

Reply:
Commented in Crowdin here: https://crowdin.com/translate/microbitorg/6426/en-nl?filter=basic&value=0#q=reference

The change was very recent.

Reply:
db5fe1f821e0ba16b8ae63ae002948f893dab130